README.md

```diff
@@ -2,7 +2,11 @@
 Embedding a video in a webpage
```

Comment:
Generally, we should improve the README with a more descriptive one. Use [this template](https://github.com/microverseinc/readme-template) as example.